### ISSUE TITLE
SPM Support

### ISSUE THREAD

**Issue by dingwilson** [dingwilson/RandomUserSwift#4]

**Comment by davidhakki** [comment]
Yes this would be great please add Swift Package Manager support

**Comment by dingwilson** [comment]
@EliteTechnicalCare - sorry I haven't had much opportunity to work on this project as of late - We definitely welcome contributions though! If you have the time to setup and add the `Package.swift`, that would be super helpful! 👍
